### PR TITLE
NTGR-622 - delete auto issuance with Ajax

### PR DESCRIPTION
**Pull request by Ndibui**
This PR updates delete auto issuance functionality to use Ajax.

> This PR builds upon PR #44
> The base of this branch will be updated to `develop` once #44 has been merged.

### CODE REVIEW

File: assets/js/accredible-dialog.js [hunk 1]
```diff
@@ -37,9 +37,37 @@ jQuery( function(){
...
+                    accredibleAjax.doAutoIssuanceAction(
+                        formData
+                    ).always(function(res){
+                        if ((typeof(res) === 'object')) {
+                            const message = res.data && res.data.message ? res.data.message : res.data;
+                            if (res.success) {
+                                closeDialog();
+                                accredibleToast.success(message, 5000);
+                                // Reload auto issuances.
+                                accredibleAjax.loadAutoIssuanceListInfo().always(function(res){
```
Comment (kei178) [inline assets/js/accredible-dialog.js:62]: I was wondering if `accredibleAjax.loadAutoIssuanceListInfo` is properly handling the pagination.

I updated all page size as `1` and tried to delete one AutoIssuance on page 3. After deleting it, I was expecting the blank third page but the first page was displayed as attached below:

Screen Shot 2022-06-20 at 10 39 04

**Review by kei178 — COMMENTED** [review]
@Ndibui - I have one question on the pagination behaviour after the delete action.

File: assets/js/accredible-dialog.js (hunk 1 above)
Comment (Ndibui) [reply assets/js/accredible-dialog.js:62]: Good catch. I should be passing `page_num` with this request.

**Review by kei178 — APPROVED** [review]
LGTM 👍

**Pull request merged**